server/src/main/java/org/elasticsearch/action/explain/ExplainResponse.java

```diff
@@ -21,19 +21,43 @@
 
 import org.apache.lucene.search.Explanation;
 import org.elasticsearch.action.ActionResponse;
+import org.elasticsearch.common.ParseField;
+import org.elasticsearch.common.bytes.BytesReference;
+import org.elasticsearch.common.collect.HppcMaps;
+import org.elasticsearch.common.document.DocumentField;
 import org.elasticsearch.common.io.stream.StreamInput;
 import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.xcontent.ConstructingObjectParser;
+import org.elasticsearch.common.xcontent.ToXContentObject;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.common.xcontent.XContentParser;
 import org.elasticsearch.index.get.GetResult;
+import org.elasticsearch.index.mapper.ParseContext;
+import org.elasticsearch.rest.action.search.RestExplainAction;
+import org.omg.CORBA.MARSHAL;
 
 import java.io.IOException;
+import java.util.Collection;
+import java.util.Map;
+import java.util.Objects;
 
 import static org.elasticsearch.common.lucene.Lucene.readExplanation;
 import static org.elasticsearch.common.lucene.Lucene.writeExplanation;
 
 /**
  * Response containing the score explanation.
  */
-public class ExplainResponse extends ActionResponse {
+public class ExplainResponse extends ActionResponse implements ToXContentObject {
```

Comment:
you could make this StatusToXContentObject and have status method return the proper status based on the exists flag